drivers/mil_blueview_driver/include/blueview_ros_driver.hpp

```diff
@@ -29,6 +32,8 @@ class BlueViewRosDriver
   cv_bridge::CvImagePtr grayscale_img, color_img;
   mil_blueview_driver::BlueViewPingPtr ping_msg;
 
+
+  double period_seconds;
```

Comment:
Private data members should have a trailing underscore, according to the [ros style guide](http://wiki.ros.org/CppStyleGuide#Member_variables), especially since you want to contribute this package. this is why auto-formatting will be great.